doc/commands/setup.rst

```diff
@@ -29,9 +38,23 @@ positional arguments
 
 
 
-.. option:: <runtime>
+.. option:: <pathogen>|<runtime>
+
+    The Nextstrain pathogen or runtime to set up.
+
+    A pathogen is usually the plain name of a Nextstrain-maintained
+    pathogen (e.g. ``measles``), optionally with an ``@<version>``
+    specifier (e.g. ``measles@v42``).  If ``<version>`` is specified in
+    this case, it must be a tag name (i.e. a release name), development
+    branch name, or a development commit id.
+
+    A pathogen may also be fully-specified as ``<name>@<version>=<url>``
+    where ``<name>`` and ``<version>`` in this case are (mostly)
+    arbitrary and ``<url>`` points to a ZIP file containing the
+    pathogen workflow contents.
```

Comment:
Having an example of what `url points to a ZIP file` looks like might be helpful here?

Reply:
Will be addressed in repush shortly.